### PR TITLE
Update contest_submission.html

### LINKED ISSUE #150: Bring thank you text in the center
On /submit_contest/ page, bring the text in the center of the page i.e., both horizontally and vertically in the center of the page.
thank you page

Also replace *Y* with *y*.

### PR DESCRIPTION
**Pull request by thomas-booker**
Centered "Thank you" and "Your contest...", and changed "... You" to "... you".

**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #150 

Centered the two texts as requested, and also changed uppercase Y to lowercase.

**Why do we need this PR?**:

To change the text, as requested.

If relevant, please include a screenshot.

**Demo (optional)**:

Some tips for you to write the instructions:
- Prefer bulleted description
- Start after checking out this branch
- Include any setup required, such as migrating databases, etc.

**Testing instructions:**

If there is any work still left to do, please add it here.

**TODOs (if any)**:

**A picture of a cute animal (not mandatory but encouraged)**:

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@thomas-booker Thanks for contributing!! 😄 As i am seeing that you are vary enthusiast to contribute in this project but as [d3vil0p3r](https://github.com/d3vil0p3r) already fixed the issue so it's fair to give hime the chance and merge his PR 🚀 .  

Hey!! This is not the end of open source development, you can look into existing [issues](https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues) and try to fix the issue. In case you stuck anywhere just message on the slack channel (Request you invite [here](https://opensourcehelp.herokuapp.com)). Anyone who are online will help you with the issues you are facing, alternatively you can search on google and stackoverflow. Maybe people had faced the same issue you faced and provided the solution.

Happy Coding!!

File: oshc/main/templates/contest_submission.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 <div class="jumbotron text-xs-center">
   <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+      <h1 class="display-3" style="text-align: center">Thank you!</h1>
+      <p class="lead" style="text-align: center"><strong>Your contest has been submitted for review.</strong></p>
```
Comment (tapaswenipathak) [inline oshc/main/templates/contest_submission.html:8]: This isn't working for me, can you please share screenshots?

Comment (thomas-booker) [reply oshc/main/templates/contest_submission.html:8]: contest submission screenshot

Comment (tapaswenipathak) [reply oshc/main/templates/contest_submission.html:8]: center of the page, like vertically and horizontally in the center of the page. I will update the description.

**Comment by tapaswenipathak** [comment]
Someone fixed this issue with 8c1d068c75437e3a2f5e46335d49c0b82278d7a6, so closing this one. Feel free to pick any other open issues or raise new.

**Pull request closed without merging**